### ISSUE TITLE
Having some trouble with vertical alignment

### ISSUE THREAD

**Issue by EnricoMonese** [treeform/pixie-python#17]
I'm trying to have some text with two lines, aligned to the bottom of some bounds, and with a small line-height so that the two lines are closer together. However the text is not going where I would expect. It looks like it's going too downwards. When aligned to bottom it even goes outside the bounds. If I leave the line-height to default it also doesn't look right.
Am I missing something?

Here's an example and it's output:

```python
import pixie

image = pixie.Image(800, 800)
image.fill(pixie.Color(1, 1, 1, 1))

paint = pixie.Paint(pixie.PK_SOLID)
paint.color = pixie.Color(1, 0, 0, 1)
ctx = image.new_context()
ctx.line_width = 1
ctx.stroke_style = paint
ctx.stroke_rect(0, 266, 800, 318)

text = "AbCd\naBcD"

font = pixie.read_font("./Inter-Bold.otf")
font.size = 182
font.line_height = 140

font.paint.color = pixie.Color(1, 0, 0, 1)
image.fill_text(
    font,
    text,
    bounds = pixie.Vector2(800, 318),
    h_align = pixie.HA_CENTER,
    v_align = pixie.VA_TOP,
    transform = pixie.translate(0, 266)
)

font.paint.color = pixie.Color(0, 1, 0, 1)
image.fill_text(
    font,
    text,
    bounds = pixie.Vector2(800, 318),
    h_align = pixie.HA_CENTER,
    v_align = pixie.VA_MIDDLE,
    transform = pixie.translate(0, 266)
)

font.paint.color = pixie.Color(0, 0, 1, 1)
image.fill_text(
    font,
    text,
    bounds = pixie.Vector2(800, 318),
    h_align = pixie.HA_CENTER,
    v_align = pixie.VA_BOTTOM,
    transform = pixie.translate(0, 266)
)

image.write_file("out.png")
```

![result](https://user-images.githubusercontent.com/19847464/154799538-041cec39-f652-44d8-9cf9-6899bf9a58d4.png)

**Comment by treeform** [comment]
Thank you for the bug, you are right it should look more like this:

![image](https://user-images.githubusercontent.com/125948/155018043-648057fe-1ac4-4124-b382-b9edd48b2a7b.png)

We will try to fix it.

**Comment by guzba** [comment]
I have looked into this and determined the issue. Things work as expected (per Figma, our source-of-truth) when the default line-height is used. Once the custom line-height of 140 is set, we do not do the correct thing. I see where so that's what needs fixing.

**Comment by guzba** [comment]
This should now be fixed in 4.0.1. Please re-open if things don't seem quite right even with the update.

**Comment by EnricoMonese** [comment]
Wow that was fast! Can confirm, it works!
